lib/main.dart

```diff
@@ -1,9 +1,29 @@
+import 'dart:convert';
+
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:movie/flavors/env_enum.dart';
 import 'package:presentation/app/my_app.dart';
+import 'flavors/config_data.dart';
+import 'flavors/config_presentation.dart';
 import 'di/app_injector.dart';
 
-void main() {
+void mainCommon({Environment env = Environment.production}) async {
   WidgetsFlutterBinding.ensureInitialized();
-  initAppInjector();
-  runApp(const MyApp());
-}
\ No newline at end of file
+  final currentEnv = env == Environment.production ? 'prod' : 'sandbox';
+  final ConfigData configData = ConfigData.fromJson(await readJson(currentEnv));
```

Comment:
move `await readJson(currentEnv)` to separate variable as it is used in several places